google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -2959,15 +2959,15 @@ public void testBucketWithBucketPolicyOnlyEnabled() throws Exception {
           Bucket.newBuilder(bucket)
               .setIamConfiguration(
                   BucketInfo.IamConfiguration.newBuilder()
-                      .setIsBucketPolicyOnlyEnabled(true)
+                      .setIsUniformBucketLevelAccessEnabled(true)
```

Comment:
This is being tested in https://github.com/googleapis/java-storage/blob/50b099d4e01418897faa82e39422d8725d20061b/google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java#L2990-L3021

Please leave these tests in place.